libevm/VM.cpp

```diff
@@ -211,6 +211,12 @@ void VM::interpretCases()
 		//
 		// Call-related instructions
 		//
+
+		CASE(CREATE_PSH)
+		{
+			m_bounce = &VM::caseCreate;
```

Comment:
It should be available only after Metropolis and generate BadInstruction before. See for example how REVERT handles this